src/App.tsx

```diff
@@ -59,7 +60,8 @@ const App = () => {
                                         <>
                                             <Switch>
                                                 <Route exact strict component={PageNotFound} path="/404" />
-                                                <Route exact strict component={Safes} path={'/'} />
+                                                <Route exact strict component={Deposit} path={'/'} />
+                                                <Route exact strict component={Safes} path={'/safes'} />
```

Comment:
What is /safes path? We switched to using "vaults"

Reply:
that route originally was mapped to '/' so I just renamed it to '/safes' to open up the default '/' route. I've changed it to be '/vaults' instead. Now when the user presses "Vaults" in the app they will be navigated to '/vaults' and see the greyed out create facilitator button